code/modules/telesci/telesci_computer.dm

```diff
@@ -301,6 +309,21 @@
 			flick(transition_icon, A)
 	qdel(src)
 
+/obj/effect/teleport_blink/container_resist(mob/living/M)
+	if(!reset_turf)
+		return
+	if(prob(50))
+		to_chat(M, "<span class='warning'>You resist the teleportation!</span>")
+		visible_message("<span class='warning'>The teleport beam collapses!</span>")
+		M.forceMove(reset_turf)
+		M.adjustToxLoss(15) // A bit of toxins damage.
```

Comment:
I don't see why the toxin damage is necessary. That way you can just spam teleport a HoS and you'll either get them teleported into your non-descript maint cave or you'll just crit them with toxin damage if you try long enough.